client/test/unitTests/Page/Menu.test.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+import { fireEvent, render, screen, act } from '@testing-library/react';
+import MiniDrawer from 'page/Menu';
+import { MemoryRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import { closeMenu, openMenu } from 'store/menu.slice';
+import { useAuth } from 'react-oidc-context';
+import store from 'store/store';
+import { closestDiv } from '../../integration/integrationTestUtils';
```

Comment:
better to use paths without `../..` notation. The convention used in lines 3, 8 is a good one. Please follow the same.

Reply:
I will need to change this
```
"modulePaths": [
    "<rootDir>/src/"
  ]
```
in `jest.config.json` to this
```
"modulePaths": [
    "<rootDir>/"
  ]
```
offsetting all imports that use the notation in the test folder, which will take some time. It is still doable though and helps with readability. Should I proceed?

Reply:
Please proceed if it is less than a day's work. Otherwise, we will push it to future work.